### DISCUSSION TITLE
LSP/TS stopped working on latest update of LazyVim?

### DISCUSSION THREAD

**Discussion by limaceous-bushwhacker** [LazyVim/LazyVim#3058] [Q&A]
Is it just me or anyone else facing the same issue?
![image](https://github.com/LazyVim/LazyVim/assets/67668779/452e55ce-b507-4cc8-94ea-011fa1a1609c)

**Comment by dpetka2001** [comment]
No problems with either LSP or TS.

**Reply by limaceous-bushwhacker** [reply]
Quite strange, it was working fine yesterday for me. How do I go about debugging this?

**Reply by dpetka2001** [reply]
You could first try with a new LazyVim default installation and see if everything works. If it does then it must be some configuration issue on your end, so try to see what changes you made that might have caused this.

**Reply by limaceous-bushwhacker** [reply]
Thanks, I've managed to fix it.
